### PR TITLE
chore(tests): add tests for converter methods

### LINKED ISSUE #277: Write tests for convert methods
Currently, the output of `convert` method is not tested, which means that buggy code can be published without noticing.

Testing the CLI is also a good starting point, see `tests/` folder for examples.

This would be nice to test the output of all convert variants (HTML, PDF, PowerPoint).

### PR DESCRIPTION
**Pull request by taiyeoguns**
## Fixes Issue

Resolves #277 

## Description

Adds tests for convert methods

## Check List (Check all the applicable boxes)

- [x] I understand that my contributions needs to pass the checks.
- [ ] If I created new functions / methods, I documented them and add type hints.
- [ ] If I modified already existing code, I updated the documentation accordingly.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by jeertmans** [comment]
Thanks for your PR @taiyeoguns! I'll review and merge this once I have some time :-)

File: tests/test_convert.py [hunk 1]
```diff
@@ -129,3 +131,31 @@ class TestConverter:
...
+    def test_pdf_converter(
+        self, tmp_path: Path, presentation_config: PresentationConfig
+    ) -> None:
+        out_file = tmp_path / "slides.pdf"
+        PDF(presentation_configs=[presentation_config]).convert_to(out_file)
+        assert out_file.exists()
+
+    def test_pdf_converter_no_presentation_config(self, tmp_path: Path) -> None:
+        with pytest.raises(IndexError):
+            PDF().convert_to(tmp_path / "slides.pdf")
```
Comment (jeertmans) [inline tests/test_convert.py:154]: Actually, nice catch @taiyeoguns! This made me realize that I should disallow having empty lists.

The error is now different:

```suggestion
    def test_converter_no_presentation_config(self) -> None:
        with pytest.raises(ValidationError):
            Converter(presentation_configs=[])
```

And you should add this import line at the top:

```python
from pydantic import ValidationError
```

**Review by jeertmans — CHANGES_REQUESTED** [review]

File: tests/test_convert.py (hunk 1 above)
Comment (taiyeoguns) [reply tests/test_convert.py]: You're welcome @jeertmans, code updated.

**Comment by jeertmans** [comment]
Looks good to me, thanks @taiyeoguns!

**Pull request merged**